Python/Sandhya/task1/show_result.py

```diff
@@ -0,0 +1,47 @@
+import json
+import argparse
+from pathlib import Path
+
+#load json contents into an object
+def get_result(filename):
+    if Path.is_file(Path(Path.cwd(),filename)):
```

Comment:
`Path` can be used as single object extension.

- use `Path(__file__).parent` for directory of file path. `Path.cwd()` would be calling path when program is called from another location.
- Use `joinpath`